frontend/simple/views/CreateGroup.vue

```diff
@@ -235,23 +84,62 @@ export default {
         await this.$store.dispatch('syncContractWithServer', hash)
       } catch (error) {
         console.error(error)
-        this.errorMsg = L('Failed to Create Group')
+        this.form.errorMsg = L('Failed to Create Group')
       }
     }
   },
   data () {
-    // TODO: wrap this in a 'form' parameter (#297)
     return {
-      groupName: null,
-      sharedValues: null,
-      changePercentage: 80,
-      memberApprovalPercentage: 80,
-      memberRemovalPercentage: 80,
-      incomeProvided: null,
-      contributionPrivacy: 'Very Private',
-      errorMsg: null,
-      // this determines whether or not to render proxy components for nightmare
-      dev: process.env.NODE_ENV === 'development'
+      form: {
+        groupName: 'alma',
+        sharedValues: null,
+        changePercentage: 80,
+        memberApprovalPercentage: 80,
+        memberRemovalPercentage: 80,
+        incomeProvided: null,
+        contributionPrivacy: 'Very Private',
+        errorMsg: null
+      },
+      ephemeral: {
+        // this determines whether or not to render proxy components for nightmare
+        dev: process.env.NODE_ENV === 'development'
+      }
+    }
+  },
+  computed: {
+    views: function () {
```

Comment:
OK, so, I haven't checked this myself yet, and I could be mistaken, but my gut tells me that:

1. This doesn't need to be a function
2. This doesn't need to be a computed property (have you tried the `data:` field?)
3. For the stuff inside of it, the `get` / `set` stuff of the computed properties is also not necessary

Reply:
1. yes it does, see the api docs: https://vuejs.org/v2/api/#computed also, it's using `this` inside to use properties defined in data, and this is how that context is bound to the right place

2. and 3.: I did try the data field, which is good for setting an initial value, but what I want to achieve is binding that keeps up-to-date when changed in either the parent or this child component. 
for this I needed to solve how to update `form.groupName` in the parent when the value in the step child component changes.
I tried numerous solutions for that:
- firing an event in the step component and manually propagating through 4 components to get here 🤾‍♀️🤾‍♂️🤾‍♀️🤾‍♂️, and
- creating createGroup's own event bus for a more direct communication between the step and the parent 🚎
that you can see in the previous commits.
both those ways seemed way more cumbersome than this solution, where I directly use the computed value's setter to update data in the parent component.
And you can't use getter/setters for data, only for computed props.

I'm still thinking about creating a little tool that automates this kind of binding and makes it less ugly (I miss React's HOCs), but apart from that, this is the simplest solution I could find. let me know if you still see ways to simplify!

Reply:
> yes it does, see the api docs: https://vuejs.org/v2/api/#computed also, it's using this inside to use properties defined in data, and this is how that context is bound to the right place

Ah you're right, I think I might have been just used to seeing it written as `views ()` instead of `views: function ()`, and subconsciously forgotten those are the same thing, lol.

OK, this looks good, I will still play with it and see if I can think of any other way, but I'm getting more convinced that you've found the correct approach.

Kudos on the `connect` function, and nice job with making this PR conform to #297. 👍